### ISSUE TITLE
Parsing metadata with multiple namespaces is not possible

### ISSUE THREAD

**Issue by lpmusicon** [odata2ts/odata2ts#174]
Hi!
I found one interesting issue while trying out the library, for simple cases it's very convenient and works as expected. I've noticed however that as soon as you introduce more than one namespace into the metadata it breaks the parser.

I've created minimal example that demonstrates the issue:
metadata.xml
```xml
<?xml version="1.0" encoding="utf-8"?>
<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
  <edmx:DataServices>
    <Schema Namespace="Namespace.Api.Models"
      xmlns="http://docs.oasis-open.org/odata/ns/edm">
      <EntityType Name="Order">
        <Key>
          <PropertyRef Name="id" />
        </Key>
        <Property Name="id" Type="Edm.Guid" Nullable="false" />
        <Property Name="orderType" Type="Namespace.Api.Enums.OrderType" Nullable="false" /> <!-- parser won't use that type and throw an error -->
      </EntityType>
    </Schema>
    <Schema Namespace="Namespace.Api.Enums" xmlns="http://docs.oasis-open.org/odata/ns/edm">
      <EnumType Name="OrderType">
        <Member Name="Example" Value="0" />
      </EnumType>
    </Schema>
  </edmx:DataServices>
</edmx:Edmx>
```

My odata2ts.config file:
```typescript
const config: ConfigFileOptions = {
  mode: Modes.qobjects,
  emitMode: EmitModes.ts,
  prettier: true,
  skipEditableModels: true,
  services: {
    metadata: {
      source: 'metadata.xml',
      output: 'odata-client'
    }
  }
}
```

I've taken a look at the source and it looks like at the moment the parsing is done on first encountered namespace and that's it. Do you plan to take cases like that into account? From what I've checked this is valid metadata xml and I'd assume the intention is to parse it as such.

Kind regards!

You can also take a look at [this codesandbox](https://codesandbox.io/p/sandbox/vibrant-leaf-jt8wzj?file=%2Fmetadata.xml%3A10%2C64)

**Comment by texttechne** [comment]
Hi @lpmusicon,

this is an absolutely valid example in regards to the OData spec.
And yes, I've got this issue on the roadmap but without any priority.

I'll try to take a look into it and report back.

**Comment by lpmusicon** [comment]
Hi @texttechne! Thank you, that's very quick turnaround. What we have implemented on our side for the time being to circumvent the issue is literally the same solution you have provided, just done at metadata generation level. 
**So to answer in short, yep that'd be sufficient for us :)**

**Comment by texttechne** [comment]
Hi @lpmusicon,

should be fixed with version `0.25.0` of `@odata2ts/odata2ts`.
To be on the safe side: It's only the solution approach which merges together the different namespaces.

Please try it out and close the issue if it has been fixed.

**Comment by lpmusicon** [comment]
Hey! It does work perfectly, thank you so much!
